docs/docs/starlark-reference/plan.md

```diff
@@ -432,15 +432,17 @@ The `files` dictionary argument accepts a key value pair, where `key` is the pat
 The instruction returns a `struct` with [future references][future-references-reference] to the ouput and exit code of the command, alongside with future-reference to the file artifact names that were generated. 
    * `result.output` is a future reference to the output of the command
    * `result.code` is a future reference to the exit code
-   *  `result.files_artifacts` is a future reference to the names of the file artifacts that were generated and can be used by the `files` property of `ServiceConfig` or `run_sh` instruction. An example is shown below:-
+   * `result.files_artifacts` is a future reference to the names of the file artifacts that were generated and can be used by the `files` property of `ServiceConfig` or `run_sh` instruction. An example is shown below:-
 
 ```python
 
     result = plan.run_sh(
         run = "mkdir -p task && cd task && echo kurtosis > test.txt",
         store = [
             "/task",
-            "/task/test.txt",
+            # using '*' will only copy the contents of the parent directory and not the directory itself to file artifact
+            # in this case, only test.txt will be stored and task directory will be ignored
```

Comment:
I find this a bit confusing since `/task` is included just above.  Can this example be updated to use two different dir names?

Reply:
The idea is to show - how `/task` and `/task/*` behaves differently so decided to have them besides each other; if this is super confusing I can divide this into 2.

Reply:
Got it.